theano/sandbox/cuda/dnn.py

```diff
@@ -2597,7 +2597,8 @@ def local_softmax_dnn_grad(node):
                 gpu_contiguous(ins[0]),
```

Comment:
I just run them in normal mode. Guillaume, can you try this?

On Wed, Apr 20, 2016 at 8:28 PM, Pascal Lamblin <notifications@github.com>
wrote:

> In theano/sandbox/cuda/dnn.py
> <https://github.com/Theano/Theano/pull/4409#discussion_r60510323>:
>
> > @@ -2597,7 +2597,8 @@ def local_softmax_dnn_grad(node):
> >                  gpu_contiguous(ins[0]),
>
> Do they pass in DebugMode, on a GPU that supports CuDNN?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Theano/Theano/pull/4409/files/3987601b5de38f1de126f5399d32da6d4788a947#r60510323>
>